src/test/java/uk/gov/hmcts/dm/controller/StoredDocumentSearchControllerTests.java

```diff
@@ -29,12 +29,12 @@ public void testValidCommandAndSearchReturn3Documents() throws Exception {
             new StoredDocument(),
             new StoredDocument());
 
-        Pageable pageable = new PageRequest(0, 2);
+        PageRequest r = new PageRequest(0, 2);
```

Comment:
I'm never a fan of single letter variable names unless they're super obvious and common like `i`. If the method grows and the scope of the variable grows it becomes less readable.